package.json

```diff
@@ -35,9 +35,9 @@
     "test": "npm run build && npm run test:fast && npm run catch-uncommitted",
     "test:local-visual": "./scripts/test-screenshots.sh",
     "test:visual": "npm run ci:screenshot -- -o .reg/current && reg-cli .reg/current __screenshots__ .reg/diff -R .reg/report.html",
-    "jest": "jest __tests__ --maxWorkers=4",
-    "eslint": "eslint src __tests__ .storybook",
-    "eslint-fix": "eslint src __tests__ .storybook --fix",
+    "jest": "jest --maxWorkers=4",
+    "eslint": "eslint src **/*.spec.js .storybook",
```

Comment:
if spec files are in the src directory I don't think you need the wildcard expansion

Reply:
Good point, I just replaced the __tests_ part without thinking about it.